### ISSUE TITLE
Finalize ufs_public_release for testing within MRweather app

### ISSUE THREAD

**Issue by fossell** [NOAA-EMC/UPP#39]
- [ ] New community build restructuring for DTC_post needs to be merged into ufs_public_release once testing is complete.
- [x] cmake mods from #34 need to be merged.
- [ ] Tests need to see if new structure is compatible with umbrella nceplibs superproject.

**Comment by fossell** [comment]
@WenMeng-NOAA - Do suggest that we merge develop into ufs_public_release branch to bring in recent mods (e.g. issues #28 #29 #32 #37) to the ufs branch?  Will those be needed for ufs app or do you prefer to keep ufs_public_release branch frozen as-is?  I will plan to merge in current develop branch to ufs_public_release unless you have a reason not to.

**Comment by WenMeng-NOAA** [comment]
These new commits are the mods for tweaking gtg(was) data, fv3HAFS, and inline post. It is optional to sync in the public release version.

**Comment by fossell** [comment]
@WenMeng-NOAA Thanks for your guidance!

**Comment by fossell** [comment]
Closing this issue since the new build structure did not go into the release. Will create new issue for the merge.
